packages/cardpay-sdk/sdk/reward-pool/base.ts

```diff
@@ -833,8 +868,7 @@ but the balance is the reward pool is ${fromWei(rewardPoolBalanceForRewardProgra
   }
   private async getRewardTokens(): Promise<string[]> {
     let cardTokenAddress = await getAddress('cardCpxd', this.layer2Web3);
-    let daiTokenAddress = await getAddress('daiCpxd', this.layer2Web3);
-    return [cardTokenAddress, daiTokenAddress];
+    return [cardTokenAddress];
```

Comment:
For some odd reason, claiming for dai was failing. Which I thought was actually working. So I just ensured I only served proofs with card token.